sigar_linux_test.go

```diff
@@ -490,4 +490,118 @@ Inter-|   Receive                                                |  Transmit
 			})
 		})
 	})
+
+	Describe("Process", func() {
+		It("GetsProcessList", func() {
+			err := os.MkdirAll(procd+"/stat", 0777)
+			Expect(err).ToNot(HaveOccurred())
+			err = os.MkdirAll(procd+"/10/", 0777)
+			Expect(err).ToNot(HaveOccurred())
+
+			procList := &sigar.ProcList{}
+			err = procList.Get()
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(len(procList.List)).To(Equal(1))
+			Expect(procList.List[0]).To(Equal(10))
+		})
+
+		It("GetsArgs", func() {
+			cmdLineFile := procd + "/10/cmdline"
+			cmdLine := "/sbin/min\x00getty\x00/dev/tty3\x00"
+			err := os.MkdirAll(procd+"/10/", 0777)
+			Expect(err).ToNot(HaveOccurred())
+			err = ioutil.WriteFile(cmdLineFile, []byte(cmdLine), 0444)
```

Comment:
It looks like all the usages of `ioutil.WriteFile` are missing an assertion. Was leaving it off unintentional?